adapters/package.json

```diff
@@ -12,6 +12,9 @@
     "build": "tsc -b ./tsconfig.build.json"
   },
   "dependencies": {
+    "ethers": "^5.4.6"
+  },
+  "devDependencies": {
     "@defichain/salmon-fetch": "0.0.0"
```

Comment:
```suggestion
    "@defichain/salmon-fetch": "0.0.0",
    "ethers": "^5.4.6"
```

`salmon-fetch` should be dependencies?

Reply:
They both need to be in `"dependencies"`, required for compilation

Reply:
yea.. saw that error during build... moving back to dep

Reply:
`ethers` need to be in dependencies too

Reply:
ahh.. yea.. fixed!!